zproject/computed_settings.py

```diff
@@ -387,6 +387,9 @@ def get_dirs(self) -> List[PosixPath]:
     "create_realm_by_ip": [
         (1800, 5),
     ],
+    "find_account_by_ip": [
+        (1600, 10),
```

Comment:
Was this meant to be 3600 or 1800?

Reply:
Ah, 3600 - good catch :smiley: We want at least 10 in the rule because FindMyTeamForm permits submitting at  most 10 emails - so it wouldn't make sense to have that always be blocked by rate limiting.